Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -154,7 +159,7 @@ public IEnumerator WaitAndStartNextRound()
         yield return new WaitForSeconds(biddingEndDelay);
         // This needs to be called after inputs are set at start the first time this is needed.
         PlayerInputManagerController.Singleton.ChangeInputMaps("FPS");
-        SceneManager.LoadScene("CraterTown");
+        SceneManager.LoadScene(currentMapName);
```

Comment:
So we set one map and that's it? No map changes within the round?

Reply:
This is the feature after some discussion between me and Fredrik resolved to.